### PR TITLE
Added A calculator project

### PR DESCRIPTION
**Pull request by LIZA1608**
# Title and Issue number 

Title :

Issue No. :

Code Stack : 

Close #<issue_no>

# Checklist:

- [] I have mentioned the issue number in my Pull Request.
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have created a helpful and easy to understand `README.md`
- [] I have updated the Index.html file for my contribution

### CODE REVIEW

**Comment by LIZA1608** [comment]
please merge it

File: apps/react/src/challenges/calculator/calculator.module.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
+
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
```
Comment (sadanandpai) [inline apps/react/src/challenges/calculator/calculator.module.css:7]: do not modify the global styles

File: apps/react/src/challenges/calculator/App.jsx [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+                <button onClick={() => setNumber('5')} className={styles.calculator__button}>5</button>
+                <button onClick={() => setNumber('6')} className={styles.calculator__button}>6</button>
+                <button onClick={() => setOperation('*')} className={styles.calculator__operator}>*</button>
+                <button onClick={() => setNumber('1')} className={styles.calculator__button}>1</button>
+                <button onClick={() => setNumber('2')} className={styles.calculator__button}>2</button>
+                <button onClick={() => setNumber('3')} className={styles.calculator__button}>3</button>
+                <button onClick={() => setOperation('+')} className={styles.calculator__operator}>+</button>
+                <button onClick={() => setNumber('0')} className={styles.calculator__button}>0</button>
+                <button onClick={() => setNumber('.')} className={styles.calculator__button}>.</button>
+                <button onClick={() => calculate()} className={styles.calculator__operator} id="total">=</button>
```
Comment (sadanandpai) [inline apps/react/src/challenges/calculator/App.jsx:205]: format the file

**Review by sadanandpai — CHANGES_REQUESTED** [review]

**Comment by sadanandpai** [comment]
image
No need of having the background around calci and also align the number display

**Comment by LIZA1608** [comment]
ok sure

**Comment by LIZA1608** [comment]
If more changes are required then please tell??

**Comment by sadanandpai** [comment]
plz add yourself to contributor list if not present already

**Comment by LIZA1608** [comment]
Can you tell me how to do that??

**Comment by LIZA1608** [comment]
> plz add yourself to contributor list if not present already

i guess i done please check

**Pull request merged**